src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -329,7 +329,7 @@
         { "command": { "action": "switchToTab", "index": 5 }, "keys": "ctrl+alt+6" },
         { "command": { "action": "switchToTab", "index": 6 }, "keys": "ctrl+alt+7" },
         { "command": { "action": "switchToTab", "index": 7 }, "keys": "ctrl+alt+8" },
-        { "command": { "action": "switchToTab", "index": 8 }, "keys": "ctrl+alt+9" },
+        { "command": { "action": "switchToTab", "index": 99999 }, "keys": "ctrl+alt+9" },

```

Comment:
will this, per chance, show up as...

![image](https://user-images.githubusercontent.com/189190/125654741-91b5ce80-e411-4e36-9f3e-0c3b995f846b.png)

Reply:
bah

Reply:
okay fine fine. I'll set it to `uint_whatever_t_max`, and special-case that resource as "switch to the last tab"

Reply:
in latest commit

![image](https://user-images.githubusercontent.com/18356694/126507500-f83e39a5-0fc6-4640-837c-d95d41e068ee.png)

(I have the `"index":99999` version in `settings.json` to prove it works)